yarn-project/pxe/src/synchronizer/synchronizer.ts

```diff
@@ -92,6 +106,8 @@ export class Synchronizer {
   protected async work(limit = 1): Promise<boolean> {
     const from = this.getSynchedBlockNumber() + 1;
     try {
+      // TODO: is getting logs redundant? see getBlocks within lmdb_archiver_store.ts
```

Comment:
updated the comment with link to https://github.com/AztecProtocol/aztec-packages/issues/3870